.github/workflows/test-external-providers.yml

```diff
@@ -5,6 +5,14 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+    paths:
+      - 'distributions/**'
```

Comment:
Actually this was removed a few weeks ago... can you drop it? we need to do this for the other workflows too - followup PR is fine :)